From: Sebastian Ramacher <sramacher@debian.org>
Date: Wed, 8 Nov 2017 21:33:54 +0100
Subject: Link libm

---
 configure.ac | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/configure.ac b/configure.ac
index 2e5c692..71843be 100644
--- a/configure.ac
+++ b/configure.ac
@@ -13,6 +13,8 @@ AC_PROG_MAKE_SET
 LT_INIT
 
 # Checks for libraries.
+AC_SEARCH_LIBS(sqrt, m)
+
 pkg_requires=""
 PKG_CHECK_MODULES(PNG, "libpng", [
   AC_DEFINE(HAVE_PNG, 1, "have libpng")
